gensim/models/doc2vec_inner.pyx

```diff
@@ -221,65 +224,67 @@ cdef unsigned long long fast_document_dmc_neg(
 
 
 cdef init_d2v_config(Doc2VecConfig *c, model, alpha, learn_doctags, learn_words, learn_hidden,
-                     train_words=False, work=None, neu1=None, word_vectors=None, word_locks=None, doctag_vectors=None,
-                     doctag_locks=None, docvecs_count=0):
+                     train_words=False, work=None, neu1=None, word_vectors=None, words_lockf=None,
+                     doctag_vectors=None, doctags_lockf=None, docvecs_count=0):
     c[0].hs = model.hs
     c[0].negative = model.negative
-    c[0].sample = (model.vocabulary.sample != 0)
+    c[0].sample = (model.sample != 0)
     c[0].cbow_mean = model.cbow_mean
     c[0].train_words = train_words
     c[0].learn_doctags = learn_doctags
     c[0].learn_words = learn_words
     c[0].learn_hidden = learn_hidden
     c[0].alpha = alpha
-    c[0].layer1_size = model.trainables.layer1_size
-    c[0].vector_size = model.docvecs.vector_size
+    c[0].layer1_size = model.layer1_size
+    c[0].vector_size = model.dv.vector_size
```

Comment:
In general, me neither. But, sometimes a property is accessed so commonly, and/or has such a dominant abbreviation, that compactness & familiarity win out, or bootstrapping an abbreviation-as-idiom is justified. Just in these 2Vec classes, we have `sg`, `hs`, `dm`, `cbow`. There are also tons of other abbreviations grandfathered-in by familiarity: `vocab`, `id`, `num`, `fxn`, & (until recently) `iter`.

Within this domain, the case for `wv` as the local idiomatic shorthand for [subattribute holding]`word_vectors` seems strong to me. It's even useful as an intentional contrast to user-chosen variables (like `word_vectors` or `wordvectors` or `word_vecs` or `wordvecs`) they might use for either their full-model, or some temporary subset of keys/vectors of interest. And, we've been using `wv` for this for a while now.

But, the parallel set of document-vectors was still under the name `docvecs` - which as two abbreviations concatenated isn't optimal, either. (Why not `document_vectors` or `doc_vecs` or even, perhaps more accurately because there's one per *tag* key not per *document*, `tag_vectors`?) 

As of this cleanup, these `docvecs` are exactly parallel to the prior `wv`: same implementing class, same set of available operations, same typical idiomatic access patterns. So the naming should be analogous. To my mind, that means one of: (a) `.wv` & `.dv`; (b) `.wordvecs` & `.docvecs`; or (c) `.word_vectors` & `.document_vectors`. Of those, (a) seems least-bad, and beneficial in advancing a compact in-library idiom.

Reply:
At a glance, (c) would be my preference.
`sg`, `hs` &co suck too (legacy of the translation from the original C code).

I don't feel strongly about this, all seem workable. CC @mpenkov thoughts?

Reply:
I think go with (a) now and open up a ticket to clean up names globally later. This PR is already large enough.